usaspending_api/awards/models.py

```diff
@@ -158,8 +158,8 @@ class Award(DataSourceTrackedModel):
                                       "blanket purchase agreement. It is used to track the contract, and any "
                                       "modifications or transactions related to it. After October 2017, it is "
                                       "between 13 and 17 digits, both letters and numbers.")
-    parent_award = models.ForeignKey('awards.Award', related_name='child_award', null=True,
-                                     help_text="The parent award, if applicable")
+    fpds_agency_id = models.TextField(db_index=True, blank=True, null=True)
```

Comment:
Agency is a low cardinality field.  Think `fpds_agency_id` and `fpds_parent_agency_id` are worth indexing?

Reply:
Might not be, I'll drop the index from these fields and add a new index for `parent_award_piid`